modules/ossm-performance-scalability.adoc

```diff
@@ -0,0 +1,79 @@
+////
+This module included in the following assemblies:
+- /v2x/customizing-installation-ossm.adoc
+////
+
+[id="ossm-performance-scalability_{context}"]
+= Performance and scalability
+
+{ProductName} strives to provide features with minimal resource overhead and aims to support very large meshes with high request rates while adding minimal latency.
```

Comment:
I think this sentence can be a stronger statement about what the product does. The words `strives` and `aims` convey a subjunctive mood in a part of the module that should be more factual in what the product is and does and what the following content will describe.